src/nvim/ui_bridge.c

```diff
@@ -180,3 +182,24 @@ static void ui_bridge_suspend_event(void **argv)
   UI *ui = UI(argv[0]);
   ui->suspend(ui);
 }
+
+static void ui_bridge_event(UI *b, char *name, Array args, bool *args_consumed)
```

Comment:
remove, add the relevant declations to ui_events.in.h and autogeneration takes care of this.